config/routes.rb

```diff
@@ -2,5 +2,9 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+  get "/users/:id/", to: "users#show"
+  get "/users/:user_id/posts/:id", to: "posts#show"
+  get "/users/:user_id/posts", to: "posts#index"
```

Comment:
- [ ] **[OPTIONAL]** I think we can improve the code here by using the alternative `resources` helper as provided by Rails.

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```

Reply:
@TedLivist Thanks for the observation. I will improve it